examples/gateways/pulsar.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Can you add an installation guide similar to other gateways at https://github.com/argoproj/argo-events/tree/master/docs/setup?